mock1/public/index.html

```diff
@@ -2,38 +2,124 @@
 <html>
   <head>
     <meta charset="utf-8">
-    <title>Welcome to Firebase Hosting</title>
+    <title>Mock 1</title>
```

Comment:
Read about mandatory (or almost mandatory) meta tags and properties and what will happen if you miss it